external-docs/docs/guides/configuration/qube-config/columns/attributes/attribute-values.md

```diff
@@ -0,0 +1,185 @@
+# Attribute values configuration
+
+This page discusses what attribute values are, where they should be used, and how they can be defined.
+
+See the [Attributes page](./index.md) for general information about attribute columns, including when to use one,
+and a discussion of the difference between [Resource](./attribute-resources.md) attributes and
+[Literal](./attribute-literals.md) attributes.
+
+> For a detailed look at attribute values configuration options, see the [Reference table](#reference) at the
+bottom of this page.
+
+## What are attribute values?
+
+Attribute values are the cell values in the attribute column of your CSV file. csvcubed will automatically generate
```

Comment:
Hmm, kind of. csvcubed will automatically generate RDF resources from the cell values....